### ISSUE TITLE
Discuss DB we are going to need

### ISSUE THREAD

**Issue by kcalderon87** [TaraisonGitHub/festa_fiesta#10]
We need to figure out what types of tables we need to implement. Like one for users and their info, list of food items needed, etc.

**Comment by qafui** [comment]
I agree. It should be one of the first steps we make. Let's discuss that today.
